ckanext/vocabulary/templates/vocabulary/read.html

```diff
@@ -8,11 +8,22 @@
 {% endblock %}
 
 {% block page_header %}
-<h1 class="module-content">{{vocab.name}}</h1>
-{% endblock %}
+  <div class="module-content">
+    <h1>{{ h.scheming_language_text(vocab.name_translated)}} </h1>
+    {% block description %}
+      {% if vocab.description_translated %}
+      <p class="description">
+        {{ h.render_markdown(h.scheming_language_text(vocab.description_translated)) }}
+        </p>
+        {% else %}
+          <p class="empty">{{ _('There is no description for this organization') }}</p>
```

Comment:
@sagargg 
I think this should be vocabulary or category instead of organization over here.

Reply:
```suggestion
          <p class="empty">{{ _('There is no description for this vocabulary') }}</p>
```